lib/dragonfly_svg/analysers/aspect_ratio_analyser.rb

```diff
@@ -0,0 +1,18 @@
+module DragonflySvg
+  module Analysers
+    class AspectRatioAnalyser
+
+      def initialize(app)
+        original = app.analysers.items[:aspect_ratio]
+        app.add_analyser(:aspect_ratio) do |content|
+          if content.mime_type == Plugin::SVG_MIME_TYPE
+            attrs = content.analyse(:svg_properties)
+            attrs[:width].to_f / attrs[:height]
+          elsif original
+            original.call(content)
+          end
```

Comment:
We would need to check if `original` is present before calling it. A simple `original.try(:call, content)` should do the trick. Will update the code.